app/blueprints/scoring/routes.py

```diff
@@ -50,8 +49,14 @@ def score(
     if not sub_criteria.is_scored:
         abort(404)
 
-    score_form = ScoreForm()
+    state = get_state_for_tasklist_banner(application_id)
+    flags_list = get_flags(application_id)
+
+    from app.blueprints.scoring.helpers import get_scoring_class
```

Comment:
Is this here to avoid a circular import?

Reply:
no, good spot